cloudbuild_docker.sh

```diff
@@ -12,58 +12,30 @@ set -o xtrace
 # https://docs.docker.com/engine/reference/commandline/manifest_create/
 sed -i 's/^{/{"experimental": "enabled",/g' ~/.docker/config.json
 
-docker manifest create gcr.io/$PROJECT_ID/static:nonroot \
-   gcr.io/$PROJECT_ID/static:nonroot-amd64 \
-   gcr.io/$PROJECT_ID/static:nonroot-arm64 \
-   gcr.io/$PROJECT_ID/static:nonroot-s390x \
-   gcr.io/$PROJECT_ID/static:nonroot-ppc64le
-docker manifest push gcr.io/$PROJECT_ID/static:nonroot
-
-docker manifest create gcr.io/$PROJECT_ID/static:latest \
-   gcr.io/$PROJECT_ID/static:latest-amd64 \
-   gcr.io/$PROJECT_ID/static:latest-arm64 \
-   gcr.io/$PROJECT_ID/static:latest-s390x \
-   gcr.io/$PROJECT_ID/static:latest-ppc64le
-docker manifest push gcr.io/$PROJECT_ID/static:latest
-
-docker manifest create gcr.io/$PROJECT_ID/base:nonroot \
-   gcr.io/$PROJECT_ID/base:nonroot-amd64 \
-   gcr.io/$PROJECT_ID/base:nonroot-arm64 \
-   gcr.io/$PROJECT_ID/base:nonroot-s390x \
-   gcr.io/$PROJECT_ID/base:nonroot-ppc64le
-docker manifest push gcr.io/$PROJECT_ID/base:nonroot
-
-docker manifest create gcr.io/$PROJECT_ID/base:latest \
-   gcr.io/$PROJECT_ID/base:latest-amd64 \
-   gcr.io/$PROJECT_ID/base:latest-arm64 \
-   gcr.io/$PROJECT_ID/base:latest-s390x \
-   gcr.io/$PROJECT_ID/base:latest-ppc64le
-docker manifest push gcr.io/$PROJECT_ID/base:latest
-
-docker manifest create gcr.io/$PROJECT_ID/base:debug-nonroot \
-   gcr.io/$PROJECT_ID/base:debug-nonroot-amd64 \
-   gcr.io/$PROJECT_ID/base:debug-nonroot-arm64 \
-   gcr.io/$PROJECT_ID/base:debug-nonroot-s390x \
-   gcr.io/$PROJECT_ID/base:debug-nonroot-ppc64le
-docker manifest push gcr.io/$PROJECT_ID/base:debug-nonroot
-
-docker manifest create gcr.io/$PROJECT_ID/base:debug \
-   gcr.io/$PROJECT_ID/base:debug-amd64 \
-   gcr.io/$PROJECT_ID/base:debug-arm64 \
-   gcr.io/$PROJECT_ID/base:debug-s390x \
-   gcr.io/$PROJECT_ID/base:debug-ppc64le
-docker manifest push gcr.io/$PROJECT_ID/base:debug
-
-docker manifest create gcr.io/$PROJECT_ID/cc:latest \
-   gcr.io/$PROJECT_ID/cc:latest-amd64 \
-   gcr.io/$PROJECT_ID/cc:latest-arm64
-docker manifest push gcr.io/$PROJECT_ID/cc:latest
-
-docker manifest create gcr.io/$PROJECT_ID/cc:debug \
-   gcr.io/$PROJECT_ID/cc:debug-amd64 \
-   gcr.io/$PROJECT_ID/cc:debug-arm64
-docker manifest push gcr.io/$PROJECT_ID/cc:debug
-
+docker_manifest() {
+  image=$1
+  from_images=""
```

Comment:
```suggestion
  local image=$1
  local from_images=""
```

Reply:
my IDE warn me drop the `local` https://github.com/koalaman/shellcheck/wiki/SC2039

Reply:
I wouldn't bother, but if you want, change the shebang to `#!/bin/bash`.

Reply:
Sure. I updated both.

Reply:
Ah, sorry. The `docker` image doesn't have `/bin/bash`. Could you revert the shebang? And let's add the `_` prefix to these variables (without `local`).

```sh
   _image=$1
   _archs=$2
   ...
```
That'll reduce the risk of variable conflicting at the global level.

Reply:
@morlay still pending

Reply:
Never mind. I fixed it.